scripts/monit_start_service.py

```diff
@@ -50,9 +64,19 @@ def start_service(service_name):
     logging.error("Monit was unable to start " + service_name)
     return 1
   else:
+    sleep_until_service_is_up(service_name)
     logging.info("Successfully started " + service_name)
     return 0
 
+def sleep_until_service_is_up(service_name):
```

Comment:
I could do that but the code where the method is called is common for Zk and Cassandra. So it would need additional ifs there anyway to pass the respective command. I could do either way!